source/internal/_curryN.js

```diff
@@ -34,7 +34,8 @@ export default function _curryN(length, received, fn) {
       }
       combinedIdx += 1;
     }
-    return left <= 0
+    var noPlaceholder = (combined.slice(0,length).every(item=>!_isPlaceholder(item)));
```

Comment:
Please don't use es6+ apis, just use es5 apis.

Reply:
Sorry, i modify it